.github/workflows/dart.yml

```diff
@@ -49,24 +58,14 @@ jobs:
           coverage: sentry
           min-coverage: 85
 
-      - name: Install webdev
-        if: runner.os != 'Windows'
-        run: dart pub global activate webdev
-
       - name: Build example
         working-directory: dart/example
         run: |
           dart pub get
           dart compile aot-snapshot bin/example.dart
 
-      - name: Build Web example
```

Comment:
Would it make run compile here using `dart compile wasm` and also keep `webdev build` for the `example_web_legacy` package?

Reply:
thats done here: https://github.com/getsentry/sentry-dart/pull/2113/files/cec7756d37472e399f4909a557955e76c20c883f#diff-f44f3e514d2ec699f9bf82bd2f0fc5721c20bcdef54a5ebcd7c641b97e39981c

so i've removed the duplicate build directly from the ci